corehq/apps/hqwebapp/templates/hqwebapp/ko_pagination.html

```diff
@@ -0,0 +1,36 @@
+{% load i18n %}
+<!-- used by corehq/apps/hqwebapp/static/hqwebapp/js/components/pagination.js -->
+<script type="text/html" id="ko-pagination-template">
+    <div class="row">
+        <div class="col-sm-5">
```

Comment:
To make this work without inline styles, remove the `display: inline-block;`, switch those divs to spans, and wrap them in a div with the `form-inline` class.